arbitrum-docs/partials/_troubleshooting-users-partial.md

```diff
@@ -85,13 +85,15 @@
 
 <p>The <em>Validators</em> are the ones responsible for the safety of the chain; i.e., making staked claims about the chain state, disputing each other, etc.</p>
 
-<p>Currently, on Arbitrum One, the Sequencer is a centralized entity maintained by Offchain Labs. Eventually, the single Sequencer will be replaced by a distributed committee of Sequencers who come to consensus on transaction ordering. This upgrade will be an improvement; we don't want you to have to trust us not to reorder your transactions. However, it also isn't <em>strictly</em> necessary for Arbitrum One to achieve its most fundamental properties.</p>
+<p><a href="https://docs.arbitrum.foundation/state-of-progressive-decentralization">Currently</a>, on Arbitrum One, the Sequencer is a centralized entity maintained by Offchain Labs. Eventually, we expect the single Sequencer to be replaced by a distributed committee of Sequencers who come to consensus on transaction ordering. This upgrade will be an improvement; we don't want you to have to trust us not to reorder your transactions. However, it also isn't <em>strictly</em> necessary for Arbitrum One to achieve its most fundamental properties.</p>
 
 <p>In other words: <em><strong>An Arbitrum Rollup chain with a centralized Sequencer could theoretically still be trustless!</strong></em></p>
 
 <p>Which is to say — the more important thing than decentralizing the Sequencer, i.e., the thing you ought to care more about — is decentralizing the <em>Validators</em>.</p>
 
-<p>Arbitrum One's Validator set is currently allowlisted; overtime, the allowlist will grow and then be removed entirely. For more info see <a href="https://developer.arbitrum.io/mainnet-beta">"Mainnet Beta"</a>.</p>
+<p>Arbitrum One's Validator set is currently whitelisted; overtime, we expect <a href="https://docs.arbitrum.foundation/">governance</a> to expand the whitelist and eventually be removed entirely. For more info see <a href="https://docs.arbitrum.foundation/state-of-progressive-decentralization">"State of Progressive Decentralization"</a>.</p>
```

Comment:
"allowlist" was selected for a [good reason](https://www.notion.so/Why-don-t-we-say-blacklist-or-whitelist-anymore-fbd38356791c41d2a9f1dbbd0c2f919e); IMO we should stick to it.

Source doc [here](https://www.notion.so/arbitrum/Are-Sequencers-the-same-entities-as-Validators-Can-a-centralized-Sequencer-act-maliciously-e-g-875d764e5f3546ce9bc11a6427159c2a)

Reply:
100%. I just changed the FAQ and also some other mentions to "whitelist" in the docs.
Thanks for flagging! 🙏